packages/fluent-theme/src/components/activity/ActivityDecorator.module.css

```diff
@@ -277,9 +277,9 @@
   display: inline-flex;
   font-size: var(--webchat-fontSizeBase100);
   font-weight: var(--webchat-fontWeightSemibold);
-  height: 14px;
+  min-height: 12.4px;
```

Comment:
Sure? Or `calc()`? 🙂

Reply:
Border being 0.8px, but it matches snapshots. Let me think on this in another PR.